### PR TITLE
more docs about DSTs and `KeyList`

### PR DESCRIPTION
**Pull request by tarakby**
Address https://github.com/onflow/cadence-lang.org/issues/114

More documentation about:
 - domain separation tags, their purpose and usage in signature verification
 - key list (`KeyList`) functions

### CODE REVIEW

**Review by turbolent — APPROVED** [review]
Thank you!

File: versioned_docs/version-1.0/language/crypto.mdx [hunk 1]
```diff
@@ -159,8 +159,8 @@ struct PublicKey {
 }
 ```
 
-`PublicKey` supports two methods `verify` and `verifyPoP`.
-`verifyPoP` will be covered under [BLS multi-signature](#proof-of-possession-pop).
+`PublicKey` supports two methods `verify` and `verifyPoP`. 
+`verify` is the [signature verification](signature-verification) function, while `verifyPoP` is covered under [BLS multi-signature](#proof-of-possession-pop).
```
Comment (turbolent) [inline versioned_docs/version-1.0/language/crypto.mdx:163]: CI flags this link is broken

**Review by turbolent — APPROVED** [review]
Thank you!

**Review by joshuahannan — APPROVED** [review]

File: versioned_docs/version-1.0/language/crypto.mdx (hunk 1 above)
Comment (turbolent) [reply versioned_docs/version-1.0/language/crypto.mdx]: Fixed by [0e21bdf](https://github.com/onflow/cadence-lang.org/pull/116/commits/0e21bdfac8e60def837b34afb37ab780412c0dd5)

**Pull request merged**